research/40.md

```diff
@@ -0,0 +1,55 @@
+Label 40 (UM40) messages are a user defined message type. They are uplinks from ground stations to the plane. According to the ICAO's Gold (Global Operational Data Link Document) 2nd edition and ICAO Doc 4444 it is a reserved message type.
+
+They take the following forms atleast from my research and messages I've seen. 
+    1. Free-text messages from the air traffic control or other ground stations to the flight crew.
+    2. Requests for weather updates or other information
+    3. Other operational or administrative messages
+
+Some example messages for each message content
+1. Free Text discussing altitude changes with ATC.
+
+FROM ATC
+"SHOWING HIWC 15NM RADIUS
+OF LOZ. 
+TOPS ARE AT FL300"
+
+FROM Crew/Pilot (5Z message label as response)
+"OS KMIA /FTM
+WE CAN GO UP TO FL340
+IF THAT LOOKS BETTER
+
+FROM ATC
+"THAT SHOULD BE GOOD
+LGT ONCL MOD TURB 
+FL300-FL380"
```

Comment:
can you add 3 backticks (`` ``` ``) on the line above and below each message so it gets rendered in a code block?

example output would be
```
FROM ATC
"SHOWING HIWC 15NM RADIUS
OF LOZ. 
TOPS ARE AT FL300"

FROM Crew/Pilot (5Z message label as response)
"OS KMIA /FTM
WE CAN GO UP TO FL340
IF THAT LOOKS BETTER

FROM ATC
"THAT SHOULD BE GOOD
LGT ONCL MOD TURB 
FL300-FL380"
```

Reply:
Ok I think I got those added, see what you think